### PR TITLE
feat(sql): ignore response rows from queries to specific tables

### PR DESCRIPTION
**Pull request by ronnathaniel**

### CODE REVIEW

File: src/events/sql.js [hunk 1]
```diff
@@ -104,7 +104,10 @@ module.exports.wrapSqlQuery = function wrapSqlQuery(queryString, params, callbac
                         rows = res;
                     }
                     eventInterface.addToMetadata(dbapiEvent, { rowCount });
-                    if (rowCount && rows instanceof Array && rows.length) {
+                    if (rowCount &&
+                        rows instanceof Array &&
+                        rows.length &&
+                        !tracer.doesContainIgnoredKey(config.getConfig().ignoredDBTables, table)) {
```
Comment (ranrib) [inline src/events/sql.js]: you've created a very big change just to check if `config.getConfig().ignoredDBTables.includes(table)`. Can you undo all changes including the packge-lock.json and simplify this PR please

**Review by ranrib — DISMISSED** [review]

File: src/events/sql.js [hunk 2]
```diff
@@ -50,9 +53,13 @@ module.exports.wrapSqlQuery = function wrapSqlQuery(queryString, params, callbac
             sqlObj.type = 'SQL-Command';
         }
 
-        const { type, table } = sqlObj;
+        const { type } = sqlObj;
+        let { table } = sqlObj;
+        if (!table) {
+            table = (sqlObj.from.length > 0 ? sqlObj.from[0].table : '');
```
Comment (haimrait) [inline src/events/sql.js]: why did you change the table?
and if you got more than 1 table?
What happened if you select from two tables?

File: src/events/sql.js [hunk 3]
```diff
@@ -30,11 +31,13 @@ module.exports.parseQueryArgs = function parseQueryArgs(arg1, arg2) {
...
  * @param {Array} params The params argument (values)
  * @param {Function} callback The callback argument (cb)
- * @param {Object} config The connection config object
+ * @param {Object} connConfig The connection config object
  * @param {string} driver The database driver type (mysql/pg/..)
  * @returns {Array} The arguments
  */
-module.exports.wrapSqlQuery = function wrapSqlQuery(queryString, params, callback, config, driver) {
+module.exports.wrapSqlQuery = function wrapSqlQuery(
+    queryString, params, callback, connConfig, driver
```
Comment (haimrait) [inline src/events/sql.js]: better change connConfig to config

File: src/events/sql.js [hunk 4]
```diff
@@ -5,6 +5,7 @@ const tracer = require('../tracer.js');
 const serverlessEvent = require('../proto/event_pb.js');
 const eventInterface = require('../event.js');
 const errorCode = require('../proto/error_code_pb.js');
+const config = require('../config.js');
```
Comment (haimrait) [inline src/events/sql.js]: const config = require('../config.js'); change it to epsagon_config

File: src/events/sql.js (hunk 2 above)
Comment (ronnathaniel) [reply src/events/sql.js]: table extraction does not work for mysql2. Using mysql it comes from sqlObj->from[]->table
extraction now supports both if first extraction fails

File: src/events/sql.js (hunk 4 above)
Comment (ronnathaniel) [reply src/events/sql.js]: revert to epsagonConfig

File: src/events/sql.js (hunk 3 above)
Comment (ronnathaniel) [reply src/events/sql.js]: revert connConfig to config 👍

**Review by haimrait — APPROVED** [review]
